src/clj/puppetlabs/services/legacy_routes/legacy_routes_service.clj

```diff
@@ -21,7 +21,14 @@
           puppet-version (get-in config [:puppet-server :puppet-version])
           ca-settings (ca/config->ca-settings (get-config))
           ca-mount (get-route (tk-services/get-service this :CaService))
-          master-mount (master-service/get-master-path config)
+          master-ns (keyword (tk-services/service-symbol
+                               (tk-services/get-service this :MasterService)))
+          master-route-config (master-core/get-master-route-config
```

Comment:
I know this has already been merged, but was curious if this functionality previously existed in the routing service and I just missed it?